website/microkit_theme/index.hbs

```diff
@@ -53,6 +53,7 @@
         <!-- MathJax -->
         <script async type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
         {{/if}}
+	<script defer data-domain="trustworthy.systems" src="https://analytics.sel4.systems/js/script.js"></script>
```

Comment:
```suggestion

        <!-- Analytics -->
	<script defer data-domain="trustworthy.systems" src="https://analytics.sel4.systems/js/script.js"></script>
```

The comment doesn't mean much but it's useful to just visually break it up from everything else.